src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -118,6 +123,16 @@ std::string Notification::toJson(const std::string& attrsFormat)
     jh.addNumber("timesSent", this->timesSent);
   }
 
+  if (this->counter > 0)
```

Comment:
Getting the counter in GET subscription requests would make sense only in the case failed max fails feature is enabled, so maybe this should be changed to:

```suggestion
  if ((this->failsCounter> 0) && (this->maxFailsLimit> 0)
```

taking also into account the names change proposed at https://github.com/telefonicaid/fiware-orion/pull/3890/files#r667784099

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/49b0c6df2c4d993b4e0fec160026fe0ac0017cb7